lib/xcake/project/hooks.rb

```diff
@@ -0,0 +1,12 @@
+require 'hooks'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.